scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -645,11 +640,6 @@ private static BuildTargetResult CompileCliSdk(BuildTargetContext c, DotNetCli d
 
             CrossgenUtil.CrossgenDirectory(c, outputDir);
 
-            // Generate .version file
-            var version = buildVersion.NuGetVersion;
-            var content = $@"{c.BuildContext["CommitHash"]}{Environment.NewLine}{version}{Environment.NewLine}";
-            File.WriteAllText(Path.Combine(outputDir, ".version"), content);
```

Comment:
I mean the .version file being laid down on disk by the installer/tarball/.zip.  Not a version file that is being uploaded to azure.